src/js/messaging/containers/Folder.jsx

```diff
@@ -334,33 +335,48 @@ export class Folder extends React.Component {
           this.props.fetchFolder(lastRequest.id, lastRequest.query);
         };
 
-        return (
-          <p>
-            Could not retrieve the folder.&nbsp;
-            <a onClick={reloadFolder}>Click here to try again.</a>
-          </p>
+        componentContent = (
+          <div className="columns">
```

Comment:
I was just curious since I couldn't find it explicitly defined within the sass src. Is it from some package?